src/components/Guilds/ProposalCard/ProposalCard.test.tsx

```diff
@@ -0,0 +1,34 @@
+import '@testing-library/jest-dom';
+import { screen } from '@testing-library/react';
+import { default as ProposalCard } from './';
+
+import { render } from '../../../utils/tests';
+
+jest.mock('hooks/Guilds/useProposal', () => ({
+  useProposal: () => ({
+    data: {
+      title: 'Proposal Title',
+      description: 'Proposal Description',
+      contentHash: '0x0',
+      endTime: { toNumber: () => 3 },
+    },
+    loading: false,

```

Comment:
I see you mocked all Swr calls with loading key. I think it should be `isValidating` instead of `loading`. 
```
export interface SWRResponse<Data = any, Error = any> {
    data?: Data;
    error?: Error;
    mutate: KeyedMutator<Data>;
    isValidating: boolean;
}
```